recipes/asdf-wcs-schemas/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "asdf-wcs-schemas" %}
+{% set version = "0.1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/asdf_wcs_schemas-{{ version }}.tar.gz
+  sha256: e1c3ef0c9a715c9fd6f0362441dc6817633a26e4136955cad954385bd3521e4e
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - setuptools-scm
+  run:
+    - asdf >=2.8.0
+    - importlib_resources >=3  # [py<39]
```

Comment:
Would be better to just unconditionally depend on `importlib_resources` and make the recipe noarch.

Reply:
Yes, done.